### DISCUSSION TITLE
求ydp031固件

### DISCUSSION THREAD

**Discussion by xianghexingbaott** [PenUniverse/PenMods#242] [你问我答]
谁有ydp031的固件可以发我一份，我x3强刷2代专业版后屏幕不显示，按压扫描头有反应（亮光），应该是固件能用但是屏幕驱动不适配，只能重刷031的包，如果谁有x3固件的话能不能发我一下

**Comment by AronnaxM** [comment]
RedbeanW 给过固件收录地址的：github.com/PenUniverse/PenMods-release/discussions/173

**Reply by xianghexingbaott** [reply]
看过了，里面没有031的固件

**Comment by 34LiuNian** [comment]
> 谁有ydp031的固件可以发我一份，我x3强刷2代专业版后屏幕不显示，按压扫描头有反应（亮光），应该是固件能用但是屏幕驱动不适配，只能重刷031的包，如果谁有x3固件的话能不能发我一下

https://1drv.ms/u/s!Ai6Nrcz52nVPhHEoP4iqBKpaLZ3f?e=5bafHY
